pages/index.vue

```diff
@@ -332,12 +312,7 @@ export default {
     min-height: 75vh;
 
     .player {
-      position: fixed;
-      top: 50%;
-      left: 0;
-      width: 100%;
-      transform: translateY(-40%);
-      display: none;
+      margin-top: 105px;
```

Comment:
I'm seeing a lot of '%' units for things like padding in this file. I feel like we should keep consistent. I haven't looked at the whole repo code yet - what is generally used or is it all inconsistent?

Reply:
Hey @DaisyyWard for margin there using px or vh and padding there using '%' - I can use '%' but since I've removed the modal, it isn't going to work the same.